### DISCUSSION TITLE
I can't edit my avatar

### DISCUSSION THREAD

**Discussion by jonah0704** [community/community#62785] [Discussions]
### Select Topic Area

Question

### Body

![1](https://github.com/community/community/assets/139465617/46bb19df-e710-4e4f-a2e6-24abb9780410)
I want to create my avatar but I can't edit it. Can anybody help me?

**Comment by dawn0123** [comment]
Did you add adblock chrome extension?
Check it! If so, you should remove it and try it again.

**Reply by jonah0704** [reply]
It works! Thank you very much!

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
